Components/HeroSlider/index.twig

```diff
@@ -0,0 +1,44 @@
+<div class="flyntComponent themeDark" is="flynt-hero-slider">
+  <script type="application/json">{{ jsonData|json_encode }}</script>
+  <div data-slider class="slider swiper-container">
+    <div class="swiper-wrapper">
+      {% for slide in slides %}
+      <div class="swiper-slide">
+        <picture class="content-picture">
```

Comment:
Class hierarchy seems to be a little weird to me, as `.content` is on the same dom level down below. The naming would imply to me that this is a child of `.content`.

Reply:
Yeah, moved it into its own context now